rpm/boss.spec

```diff
@@ -1,15 +1,20 @@
 Name: boss
-Version: 0.9.2
+Version: 0.10.0
 Release: 1
 Summary: Build Orchestration Server System
+URL: https://github.com/MeeGoIntegration/boss
 Group: Productivity/Networking/Web/Utilities
-License: GPL2
+License: GPL-2.0-only
 Source0: boss-%{version}.tar.gz
-Requires: rabbitmq-server >= 1.7.2, python-boss-skynet > 0.6.0
```

Comment:
Isn't skynet needed to manage boss at runtime?

Reply:
Well, yes you can use 'skynet start/stop/reload' etc to manage it, but it is not strictly required. They are mostly wrappers around supervisorctl functionaly... I was actually pondering about moving this to systemd service, but the supervisor process name is used to decide things, so didn't go into that yet.